src/test/java/com/qaprosoft/carina/demo/phpTravels/LoginPageTest.java

```diff
@@ -0,0 +1,92 @@
+package com.qaprosoft.carina.demo.phpTravels;
+
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.PageOpeningStrategy;
+import com.qaprosoft.carina.demo.phpTravels.components.*;
+import com.qaprosoft.carina.demo.phpTravels.pages.BookingsPage;
+import com.qaprosoft.carina.demo.phpTravels.pages.DashboardPage;
+import com.qaprosoft.carina.demo.phpTravels.pages.DropdownNav.SettingsPage;
+import com.qaprosoft.carina.demo.phpTravels.pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginPageTest implements IAbstractTest {
+
+    @Test
+    public void testForgotAccButton() {
+        LoginPage loginPage = new LoginPage(getDriver());
+        loginPage.setPageOpeningStrategy(PageOpeningStrategy.BY_URL);
+        loginPage.open();
+        loginPage.clickForgotAcc();
+        loginPage.typeResetEmail("user@jstravels.org");
+        loginPage.clickResetButton();
+        Assert.assertEquals(loginPage.getEmailNotFoundText(), "Email Not Found");
+    }
+
+    @Test
+    public void testLoginAdminAcc() {
+        loginIn("admin@phptravels.com", "demoadmin");
+        DashboardPage dashboardPage = new DashboardPage(getDriver());
+        Assert.assertTrue(dashboardPage.isPageOpened());
+    }
+
+    @Test
+    public void testChangeName() {
+        loginIn("admin@phptravels.com", "demoadmin");
+        DashboardPage dashboardPage = new DashboardPage(getDriver());
+        LeftMenuBar leftMenu =dashboardPage.getLeftMenuBar();
+
+        SettingsPage settingsPage = leftMenu.openSettingsSubmenu();
+        settingsPage.typeNameText("JS Travels | Travel Technology Partner");
+        settingsPage.clickSaveChangeButton();
+        Assert.assertTrue(settingsPage.getChangesSavedText());
+    }
+
+    @Test
+    public void testUnpaidBookingsMenu() {
+        loginIn("admin@phptravels.com", "demoadmin");
+        DashboardPage dashboardPage = new DashboardPage(getDriver());
+        NavigationBar navigationBar = dashboardPage.getNavigationBar();
+
+        BookingsPage bookingsPage = navigationBar.openBookingsPage();
+        BookingsMenu bookingsMenu = bookingsPage.getBookingsButtons();
+        bookingsMenu.clickUnpaidBookings();
+        Table tableChosen = bookingsPage.getBookingsTable();
+        String bookingStatus = tableChosen.getChosenPaidStatus("1").toUpperCase();
```

Comment:
in your test case I see that you have to verify that every row, not the only one

Reply:
for this purpose "for" loop is great
you can create method in table component `int getRowsCount();`
then in test implement loop
```
for (int i = 0; i < table.getRowsCount(); i++) {
// assertion here
}
```